.pipelines/cni/lsg/lsg-cni-intergration-template.yaml

```diff
@@ -0,0 +1,179 @@
+parameters:
+  dependsOn: ""
+  name: ""
+  clusterType: ""
+  clusterName: ""
+  nodeCount: ""
+  vmSize: ""
+  os: linux
+  arch: ""
+  osSKU: Ubuntu
+
+# CNIv2
+# + Should be able to scale up/down the pods successfully certain number of times.
+# + Node reboot scenarios should be covered.
+# + The CNS state should be validated with that of CNI state.
+# + Pods should have ips assigned and connectivity/datapath test should be present.
+# + CNS restart and validates the state
+
+stages:
+  - stage: create_${{ parameters.name }}
+    variables:
+      commitID: $[ stagedependencies.setup.env.outputs['SetEnvVars.commitID'] ]
+    pool:
+      name: $(BUILD_POOL_NAME_DEFAULT)
+    dependsOn:
+      - setup
+    displayName: "Create Cluster - ${{ parameters.clusterName }}"
+    jobs:
+      - job: create_aks_cluster_with_${{ parameters.name }}
+        steps:
+          - task: AzureCLI@1
+            inputs:
+              azureSubscription: $(BUILD_VALIDATIONS_SERVICE_CONNECTION)
+              scriptLocation: "inlineScript"
+              scriptType: "bash"
+              addSpnToEnvironment: true
+              inlineScript: |
+                make -C ./hack/aks ${{ parameters.clusterType }} \
+                AZCLI=az REGION=$(LOCATION) SUB=$(SUB_AZURE_NETWORK_AGENT_BUILD_VALIDATIONS) \
+                CLUSTER=${{ parameters.clusterName }}-$(commitID) \
+                VM_SIZE=${{ parameters.vmSize }} \
+                AUTOUPGRADE=none
```

Comment:
Will this be an issue/ flagged in s360 ? 
Given the pipeline runs for 4hrs ?

Reply:
AUTOUPGRADE is an AKS feature that triggers once a week unless something major needs to be done. This prevents any unintended upgrades during testing as this pipeline will be triggered remotely by LSG pipelines. To ensure that it does not trigger s360, clusters are cleaned up immediately.

Also, pipeline only takes 1hr 30 mins to run 💨 .